redisinsight/ui/src/utils/formatters/valueFormatters.tsx

```diff
@@ -108,16 +110,30 @@ const formattingBuffer = (
       }
     }
     case KeyValueFormat.Vector32Bit: {
+      const utfVariant = bufferToUTF8(reply)
       try {
+        if (props?.isField) return { value: utfVariant, isValid: true }
```

Comment:
`isField` - is not common name. I suggest call this argument like `skipVector`

Reply:
done